template/src/api/authSlice.test.ts

```diff
@@ -63,45 +40,13 @@ describe('#watchAuthTokens', () => {
   })
 })
 
-describe('#watchLogInSaga', () => {
-  it('should log user in and put success action with tokens', () => {
-    return expectSaga(watchLogInSaga)
-      .provide([[matchers.call.fn(logIn), fakeAuthTokens]])
-      .put(logInAsyncSuccess(fakeAuthTokens))
-      .dispatch(logInAsync(fakeCredentials))
-      .silentRun()
-  })
-
-  it('should put log in error action wit error message on auth error', () => {
-    return expectSaga(watchLogInSaga)
-      .provide([[matchers.call.fn(logIn), throwError(new Error(logInErrorMessage))]])
-      .put(logInAsyncFailure(logInErrorMessage))
-      .dispatch(logInAsync(fakeCredentials))
-      .silentRun()
-  })
-})
-
 describe('#authSlice', () => {
   const initialState = authSlice.getInitialState()
 
-  describe('#loginAsync', () => {
-    it('should change tokens state to Loading', () => {
-      const state = authSlice.reducer(initialState, logInAsync(fakeCredentials))
-      expect(state.tokens).toEqual(Loading)
-    })
-  })
-
   describe('#logInAsyncSuccess', () => {
     it('should store auth tokens', () => {
       const state = authSlice.reducer(initialState, logInAsyncSuccess(fakeAuthTokens))
-      expect(state.tokens).toEqual(Success(fakeAuthTokens))
-    })
-  })
-
-  describe('#logInAsyncFailure', () => {
-    it('should store auth error message', () => {
-      const state = authSlice.reducer(initialState, logInAsyncFailure(logInErrorMessage))
-      expect(state.tokens).toEqual(Failure(logInErrorMessage))
```

Comment:
No need to test switching states: `Loading`, `Success` and `Failure`, because it's handled by `useQuery`/`useMutation` in library